src/Rules/PHPUnit/DataProviderDeclarationRule.php

```diff
@@ -61,6 +51,9 @@ public function getNodeType(): string
 		return Node\Stmt\ClassMethod::class;
 	}
 
+	/**
+	 * @param Node\Stmt\ClassMethod $node
```

Comment:
This isn't needed because there's already `@implements Rule<Node\Stmt\ClassMethod>` above the class. Latest PhpStorm already understands this.

Reply:
Indeed. Removed.
Thanks